metricbeat/module/mysql/performance/_meta/fields.yml

```diff
@@ -26,6 +26,9 @@
         - name: 'avg.timer.wait'
           type: long
           description: Average wait time of the summarized events that are timed
+        - name: 'schemaname'
```

Comment:
Do you think that the `schema` would be more relevant field name here?

Reply:
Yes, we can do that. Thanks!

Reply:
While doing schemaname to schema we are getting the following error. We can keep schemaname to avoid this error.

Reply:
Which error?

Reply:
Updated